### PR TITLE
Multiple permissions fixes for projects

### PR DESCRIPTION
**Pull request by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf**
As communicated in `discord/develop`, here's this little PR.
I believe there is no reason to show the 'New Project board' button for users that are not signed in.
**Before:**
before
**After:**
after

### CODE REVIEW

File: templates/repo/projects/view.tmpl [hunk 1]
```diff
@@ -11,7 +11,9 @@
 			</div>
 			<div class="column right aligned">
 				{{if .PageIsProjects}}
-				<a class="ui green button show-modal item" data-modal="#new-board-item">{{.i18n.Tr "new_project_board"}}</a>
+					{{ if $.IsSigned }}
```
Comment (lafriks) [inline templates/repo/projects/view.tmpl]: It would be better to add this condition to previous if:
```go
{{if and $.IsSigned .PageIsProjects }}
```

**Review by lafriks — CHANGES_REQUESTED** [review]

**Review by 6543 — APPROVED** [review]

**Review by zeripath — COMMENTED** [review]
This is not the correct test - we should only show the button if the user can create project boards

**Comment by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf** [comment]
> This is not the correct test - we should only show the button if the user can create project boards

This should now be addressed by the latest changes.
Button is not shown for unauthorized users nor if the repo is archived.

**Comment by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf** [comment]
projectboardmenu
Only show this dropdown menu to authorized users (same as the button).
as per f60df0d

**Review by lafriks — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**